src/main/java/com/rocketden/main/socket/WebSocketConnectionEvents.java

```diff
@@ -81,6 +88,14 @@ public void onSocketConnected(SessionConnectedEvent event) {
             Room room = user.getRoom();
             RoomDto roomDto = RoomMapper.toDto(room);
             socketService.sendSocketUpdate(roomDto);
+
+            // If a game exists, update the room info for that game
+            try {
+                Game game = gameService.getGameFromRoomId(room.getRoomId());
+                game.setRoom(room);
+                game.getPlayers().get(user.getUserId()).setUser(user);
+                socketService.sendSocketUpdate(GameMapper.toDto(game));
+            } catch (ApiException ignored) {}
```

Comment:
This does have a Sonarlint warning (see below image), but I'm ok with that -- I'm wondering, would it be useful to add any logging statement there?

<img width="594" alt="Sonarlint Warning on Empty Code Block" src="https://user-images.githubusercontent.com/7987279/110225214-0b05f280-7e98-11eb-8bb9-395a6d1f5969.png">

Reply:
Also, since you have this same block of code in two places (here and the method below), perhaps you make another method it for this code and invoke that?

Reply:
Sounds good - just made a new method for this and also added our first log for the main repo haha (which would prob be a good habit from here on out)